src/services/services.ts

```diff
@@ -3798,6 +3798,10 @@ module ts {
                 // Try getting just type at this position and show
                 switch (node.kind) {
                     case SyntaxKind.Identifier:
+                        if(isLabelName(node)){

```

Comment:
I don't feel like this should be necessary.  If this node has a type, we'd want to display it.